client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -182,6 +182,8 @@ const Header: FunctionComponent = () => {
 				return;
 			}
 			resetOnboardStore();
+			// TODO: remove this :)
+			console.log( 'Call `calypso_signup_complete` here?' );
```

Comment:
That's where I was thinking we'd call it :)

Reply:
Was just testing the `/new` flow and noticed that `calypso_signup_complete` is called before checkout, namely because [all the steps in the prelaunch flow](https://github.com/Automattic/wp-calypso/blob/9793440/client/signup/config/flows-pure.js#L358) (all one of them) are satisfied **after** the plans step.  

<img width="1077" alt="Screen Shot 2020-04-16 at 7 45 10 pm" src="https://user-images.githubusercontent.com/6458278/79442146-814cfc80-801b-11ea-894f-720798fd9561.png">

I guess that won't matter much so long as we use whatever flow id we use for `calypso_signup_complete` in Gutenboarding for our `calypso_signup_complete`  call.